src/modules/notebook-panel/Overview.js

```diff
@@ -14,26 +14,46 @@ import {Form, useFormHook} from '../form';
 import ImagesOverview from '../images/ImagesOverview';
 import MeasurementsOverview from '../measurements/MeasurementsOverview';
 import NotesOverview from '../notes/NotesOverview';
+import OtherFeaturesOverview from '../other-features/OtherFeaturesOverview';
 import SamplesPage from '../samples/SamplesPage';
 import {editedSpotProperties} from '../spots/spots.slice';
 import {TagsAtSpotList} from '../tags';
+import ThreeDStructuresOverview from '../three-d-structures/ThreeDStructuresOverview';
 import notebookStyles from './notebookPanel.styles';
 
 const Overview = props => {
   const dispatch = useDispatch();
   const spot = useSelector(state => state.spot.selectedSpot);
   const [isTraceSurfaceFeatureEnabled, setIsTraceSurfaceFeatureEnabled] = useState(false);
   const [isTraceSurfaceFeatureEdit, setIsTraceSurfaceFeatureEdit] = useState(false);
+  const [refresh, setRefresh] = useState(false);
   const formRef = useRef(null);
   const [useForm] = useFormHook();
+  const featuresData = useSelector(state => state.spot.selectedSpot?.properties?.other_features || []);
```

Comment:
You can get this off `spot` above so we don't need a separate `useSelector` for all each section of data in the Overview. We don't even actually need the data either just to know if there is data or not.